kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,18 +1,22 @@
 return {
   fields = {
-    timeout = {type = "number", default = 60000, required = true },
-    keepalive = {type = "number", default = 60000, required = true },
-    aws_key = {type = "string", required = true},
-    aws_secret = {type = "string", required = true},
-    aws_region = {type = "string", required = true, enum = {
+    timeout = { type = "number", default = 60000, required = true },
```

Comment:
It was explicit request from @thibaultcha. I would not ever change this because I agree that PR should be for single purpose.